changelog/release-support-lifecycle.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: RisingWave release and support lifecycle
+sidebarTitle: Release and support lifecycle
+description: "This document provides an overview of the RisingWave versioning scheme, release cadence, and a summary of the support lifecycle for different versions. It also links to detailed policies for both version and feature deprecation."
+---
+
+This page provides a high-level overview of how RisingWave versions are numbered, how frequently releases occur, and how long different versions are supported. It serves as a central point for understanding the overall release and support process.
+
+## Versioning scheme (semantic versioning)
+
+RisingWave adheres to the [Semantic Versioning (SemVer)](https://semver.org/) standard, using a three-part numbering system: `Major.Minor.Patch`.
```

Comment:
I don't think our major version part follows SemVer. There's no breaking change between 1.10 and 2.0, it's more for branding. We are actually using minor version for the functionality of major version in SemVer. cc @xxchan